builtin_interfaces/Time start_time

uint8 order
std_msgs/Float32MultiArray[] coeff
float32[] durations
int8[] directions